data/headerNavLinks.ts

```diff
@@ -4,6 +4,34 @@ const headerNavLinks = [
   { href: '/tags', title: 'Tags' },
   { href: '/projects', title: 'Projects' },
   { href: '/about', title: 'About' },
+  {
+    title: 'Dropdown',
+    children: [
+      { href: '/projects', title: 'Projects' },
+      { href: '/about', title: 'About' },
+    ],
+  },
 ]
 
+type NavItemBase = {
```

Comment:
Prefer to move over the types to `NavOptions.tsx`

Reply:
Moved to NavOptions